### DISCUSSION TITLE
[预览]v1.12.6

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#1780] [分享 / Share]
- 下载视频的导出支持其他脚本扩展按钮. (#1762) (文档: https://github.com/the1812/Bilibili-Evolved/blob/preview/doc/download-video-extensions.md)
- `自定义顶栏`的排行入口支持更细分的链接. (#1291)
- 修复`夜间模式计划时段`在系统夜间后进入网页无法立即开启夜间的问题. (#1768)
- 修复`稍后再看重定向`在稍后再看列表页面有时失效的问题. (#1773)
- `默认播放器模式`和`自动定位到播放器`整合了去除b站宽屏的网页滚动的效果. (#483)
- 夜间模式改进.
- `简化评论区`改进: (主要是针对那个新版评论区, 之前在`快速收起评论区`里的相同代码已移除)
  - 表情选择按钮移动到了发送按钮的下方.
  - 点赞/踩能够以主题色点亮了.
  - 吸附的评论回复区:
    - 背景更换为半透明渐变.
    - 样式与顶上那个固定的回复区统一.
    - 能够响应页面宽度变化.
    - 吸附到评论区底端而不是整个页面的底端.

![image](https://user-images.githubusercontent.com/26504152/116000065-5085a700-a621-11eb-81ca-d437338887aa.jpg)

<details>
<summary><strong>附赠: 新版评论区迷惑行为大赏</strong></summary>

![image](https://user-images.githubusercontent.com/26504152/116000087-5f6c5980-a621-11eb-8071-3273957ad01c.jpg)
![image](https://user-images.githubusercontent.com/26504152/116000092-65fad100-a621-11eb-9fc0-c11c47b99fc2.jpg)
![image](https://user-images.githubusercontent.com/26504152/116000098-6abf8500-a621-11eb-9778-a979d94ebda5.jpg)
![image](https://user-images.githubusercontent.com/26504152/116000100-6e530c00-a621-11eb-9cf7-da0dcb8767f7.jpg)

</details>

👍**感谢 PR 作者**
- [xxxuuu](https://github.com/xxxuuu): PR #1758
- [FoundTheWOUT](https://github.com/FoundTheWOUT): PR #1767

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.6-preview'>[预览]v1.12.6</a>.</em>

**Comment by Geeyun-JY3** [comment]
表情选择按钮放在发送按钮的下方会不会容易误触啊？
类似于 GitHub 的 `Close issue` 和 `Comment`，这样上下的相对位置我觉得更容易误触。

**Reply by the1812** [reply]
先用着看看 太容易误触了就再改改 反正是预览版(

**Comment by maocd008** [comment]
希望可以将【整合了去除b站宽屏的网页滚动的效果】分离出来作为单独的开关，而不是整合进来
因为在1680x1050分辨率下，我就没使用过自动定位，只靠自动宽屏+B站自带的网页滚动效果就能正好居中在屏幕中心，既能看到标题也能看到硬币收藏功能栏，效果很好
但是更新1.12.6预览之后，因为整合了关闭网页滚动的效果，播放器被下拉了一些，必须使用自动定位才能变回原来的位置，但在点击其他视频时自动定位就会失效，反倒体验没有原来好了
我现在迫不得已使用了播放器置顶功能，放弃了UP主信息和三联功能栏

**Reply by the1812** [reply]
可以的
